### ISSUE TITLE
Editor not loading when debug flag is on

### ISSUE THREAD

**Issue by mponizil** [mixpanel/mixpanel-js#127]
Currently getting a 404 for http://mixpanel.com/site_media/compiled/reports/collect-everything/editor.js?_ts=1490288203125 while http://mixpanel.com/site_media/bundle-webpack/reports/collect-everything/editor.min.js?_ts=1490368172838 responds successfully.

**Comment by jbwyme** [comment]
Yep this is a bug for sure, I'll get a fix up and follow-up here afterwards.

**Comment by jbwyme** [comment]
Shipped an update in v2.12.0 that should fix the problem. Please confirm and close, thanks!
